templates/Gemfile.erb

```diff
@@ -30,7 +30,7 @@ end
 
 group :development, :test do
   gem "bundler-audit", require: false
-  gem "byebug"
+  gem "pry-byebug"
```

Comment:
Do we want to maintain the alphabetical ordering convention thoughtbot's version uses?